nicsdru_nidirect_theme.theme

```diff
@@ -17,6 +17,21 @@ use Drupal\user\Entity\User;
 use Drupal\Component\Utility\Html;
 use Drupal\taxonomy\Entity\Term;
 
+/**
+ * Implements hook_page_attachments_alter().
+ */
+function nicsdru_nidirect_theme_page_attachments_alter(array &$page) {
+
+  // Attach layout builder styles for authenticated users.
+  $userCurrent = \Drupal::currentUser();
+
+  if ($userCurrent->isAuthenticated()) {
+    $page['#attached']['library'][] = 'nicsdru_nidirect_theme/authenticated-styles';
+  }
+
+  return $page;
```

Comment:
You shouldn't have to return `$page` as it's given by reference rather than by value, so any changes to $page should directly affect the original instance of the variable.

Reply:
Yep - will remove that.